spec/lang/well-formed.md

```diff
@@ -523,12 +523,17 @@ impl PlaceExpr {
             }
             Field { root, field } => {
                 let root = root.check_wf::<T>(locals, prog)?;
-                let (_offset, field_ty) = match root {
-                    Type::Tuple { sized_fields, unsized_field: Some(unsized_field), .. } if field == sized_fields.len() => unsized_field,
+                let field_ty = match root {
+                    Type::Tuple { sized_fields, unsized_field, .. } if field == sized_fields.len() => {
```

Comment:
Please add a FIXME saying we'd like to match `unsized_field` on `Some` here but specr currently does not support that.